invary_runtime_integrity/README.md

```diff
@@ -0,0 +1,44 @@
+# Invary Runtime Integrity
+
+Visualize the Runtime Integrity of your Invary managed operating systems.
+
+## Overview
+
+Invary validates the Runtime Integrity of operating systems and detects rootkits that can deceive other systems. This integration comes with an out-of-the-box dashboard that visualizes your Runtime Integrity, providing confidence in your runtime security or spotlighting endpoints that lack integrity and may be compromised. The Invary dashboard also provides insights into your operating system inventory at runtime, including distributions and kernel versions in use. 
+
+
+This integration uses the [Invary API][1].
+
+## Setup
+
+- Once this integration has been uninstalled, any previous authorizations are revoked. 
+- Additionally, ensure that all API keys associated with this integration have been disabled by searching for the integration name on the [API Keys page][3].
```

Comment:
Can you create an `### Uninstallation` section, and add this under there? This will live underneath `### Validation`.

Reply:
Updated